testing/generate.js

```diff
@@ -135,8 +135,14 @@ export class RetoolApplication {
     // Click [data-testid="open-tests-modal"]
     await this.page.click('[data-testid="open-tests-modal"]')
 
+    // wait for page to load
+    await this.page.waitForLoadState()
```

Comment:
Can wait for 'load', 'domcontentloaded', or 'networkidle' (https://playwright.dev/docs/api/class-page/#page-wait-for-load-state, defaults to 'load'). Using 'load' but 'networkidle' could also work since requests are fired during the execution of tests if the app has queries

Setting timeout to 0 to set no timeout